ctest.h

```diff
@@ -16,6 +16,10 @@
 #ifndef CTEST_H
 #define CTEST_H
 
+#if !(defined (_GNU_SOURCE) || defined (_BSD_SOURCE) || defined(_POSIX_C_SOURCE))
+#undef CTEST_SEGFAULT
```

Comment:
Windows isn't one of those modern systems anyway, so if we want to support it in the future then `kill` must go (but not by disabling the whole segfault feature) IMHO :)